### PR TITLE
Performance reporting

### PR DESCRIPTION
**Pull request by lastminutediorama**
This adds a CLI command for generating and outputing some reports based on any SQL queries we want to use to gather metrics.

The intention is to use Celery Beat and/or cron to generate reports every week which can call this command, which will also send the output to a designated email address.

### CODE REVIEW

File: src/planscape/planning/management/commands/performance_report.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+
+class Command(BaseCommand):
+    help = "Runs some SQL reports about performance and either emails them or outputs to stdout"
+
+    message_text = ""
+    emailing = False
+    recipient = None
+
+    query_folder = "planning/report_queries/"
```
Comment (pflopez) [inline src/planscape/planning/management/commands/performance_report.py:17]: we should add this to gitignore

File: src/planscape/planning/management/commands/performance_report.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                        formatted_row = "  ".join(
+                            str(value).ljust(width)
+                            for value, width in zip(row, max_widths)
+                        )
+                        self.write(formatted_row)
+                else:
+                    self.write("** No rows **")
+
+            except Exception as e:
+                self.stderr.write("Error executing SQL: {}".format(e))
```
Comment (pflopez) [inline src/planscape/planning/management/commands/performance_report.py:74]: Looks like we are not capturing any of the exeption errors while sending the email (they are going out on stderr).
Not sure what the email will look in this case. Just empty? Do we wanna catch that somehow, just saying like "there was an error, etc et " on the email vs empty ?

File: src/planscape/planscape/settings.py [hunk 3]
```diff
@@ -391,3 +392,5 @@
 CRONJOBS = [
     ("0 0 * * *", "planning.cron.delete_old_shared_links"),  # Runs at midnight daily
 ]
+
+REPORT_RECIPIENT_EMAIL = "no-reply@planscape.org"
```
Comment (pflopez) [inline src/planscape/planscape/settings.py:396]: the nittiest of nits, but I think we we have this email already on a couple of places. Maybe we can have a NO_REPLY constant and use that instead?

File: src/planscape/planning/management/commands/performance_report.py (hunk 2 above)
Comment (lastminutediorama) [reply src/planscape/planning/management/commands/performance_report.py:74]: That's a great point. I added some output that will get included to the email.

File: src/planscape/planning/management/commands/performance_report.py (hunk 1 above)
Comment (lastminutediorama) [reply src/planscape/planning/management/commands/performance_report.py:17]: Oh, maybe it's not clear, but this folder actually includes the SQL queries that we use for reporting, so this should be in version control. See the SQL query files in this PR.

File: src/planscape/planscape/settings.py [hunk 4]
```diff
@@ -393,4 +393,6 @@
     ("0 0 * * *", "planning.cron.delete_old_shared_links"),  # Runs at midnight daily
 ]
 
-REPORT_RECIPIENT_EMAIL = "no-reply@planscape.org"
+REPORT_RECIPIENT_EMAIL = config(
+    "REPORT_RECIPIENT_EMAIL", default="no-reply@planscape.org"
```
Comment (pflopez) [inline src/planscape/planscape/settings.py:397]: I was thinking that this could be `DEFAULT_FROM_EMAIL`

**Review by pflopez — APPROVED** [review]
LGTM!

**Pull request merged**